src/tests/signatures.rs

```diff
@@ -419,6 +419,19 @@ fn test_hmac_signatures() {
     );
     assert_eq!(ret, CKR_OK);
 
+    #[cfg(feature = "pkcs11_3_2")]
+    {
+        /* check SignatureVErify API too */
+        let ret = sig_verifysig(
+            session,
+            key_handle,
+            &mut testcase.value,
+            &mut testcase.result,
+            &mut mechanism,
+        );
+        assert_eq!(ret, CKR_OK);
+    }
+
```

Comment:
Given the same API of the `sig_verifysig()` and `sig_verify()`, should we wrap the SigVerify into the first test function (`sig_verify()` to test both of the APIs, instead of adding it to each invocation of `sig_verify()` with the `cfg(` condition?

Or is there still some mechanism that does not implement this new API?